CHANGELOG.md

```diff
@@ -4,6 +4,48 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.75.0
+
+### Important Changes
+
+- **feat(opentelemetry): Add new `@sentry/opentelemetry` package (#9238)**
+
+This release publishes a new package, `@sentry/opentelemetry`. This is a runtime agnostic replacement for `@sentry/opentelemetry-node` and exports a couple of useful utilities which can be used to use Sentry together with OpenTelemetry.
+
+You can read more about [@sentry/opentelemetry in the Readme](https://github.com/getsentry/sentry-javascript/tree/develop/packages/opentelemetry).
+
+- **feat(replay): Allow to treeshake rrweb features (#9274)**
+
+Starting with this release, you can configure the following build-time flags in order to reduce the SDK bundle size:
+
+* `__RRWEB_EXCLUDE_CANVAS__`
+* `__RRWEB_EXCLUDE_IFRAME__`
+* `__RRWEB_EXCLUDE_SHADOW_DOM__`
+
+You can read more about [tree shaking in our docs](https://docs.sentry.io/platforms/javascript/configuration/tree-shaking/).
+
```

Comment:
l: we should mention that this is about Replay. I don't think everyone knows that rrweb is a dependency of replay.